data/_models/shared/Collection events.csv

```diff
@@ -1,9 +1,9 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
-Collection events,,,,,,,,,,,dcterms:PeriodOfTime,Definition of a data collection event for a resource,DataCatalogueFlat,
```

Comment:
If collection is dct:inSeries, this should be dct;Dataset. I think we can lose dct:PeriodOfTime

Reply:
`dct:PeriodOfTime` is a weird mistake, I think it should be `dct:Dataset`.
Then the next line makes sense in my view as it states:
```[collection event] is part of/in series [collection]```